packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -718,6 +725,37 @@ export default util.createRule<Options, MessageId>({
         return;
       }
 
+      // nullable enum
+      if (is('nullish', 'number', 'enum')) {
+        if (!options.allowNullableEnum) {
+          if (isLogicalNegationExpression(node.parent!)) {
+            // if (!nullableEnum)
```

Comment:
Are these leftover from somewhere?

```suggestion
```

Reply:
No, these are intentional leftover just like those in other parts.
https://github.com/kazizi55/typescript-eslint/blob/add-allowNullableEnum-to-strict-boolean-expressions/packages/eslint-plugin/src/rules/strict-boolean-expressions.ts#L620

https://github.com/kazizi55/typescript-eslint/blob/add-allowNullableEnum-to-strict-boolean-expressions/packages/eslint-plugin/src/rules/strict-boolean-expressions.ts#L701

Shoud I remove the leftover?

Reply:
Yeah, I'd say remove it. We're not big on leaving old commented out code - it tends to get out of date over time and make the file harder to read through.